test/transactions.test.js

```diff
@@ -1,7 +1,8 @@
 /* eslint-disable no-unused-expressions */
 import '@babel/polyfill';
-import chai, { expect } from 'chai';
+import chai, { expect, assert } from 'chai';
 import chaiHttp from 'chai-http';
+import sinon from 'sinon';
```

Comment:
what do you `sinon` for?

Reply:
I used it to mock 500 errors and an API call